e2e/test/scenarios/sharing/pulse.cy.spec.js

```diff
@@ -11,22 +11,40 @@ describe("scenarios > pulse", { tags: "@external" }, () => {
   });
 
   it("should create a new pulse", () => {
+    const pulseTitle = "Foo";
+
+    cy.intercept("GET", "/api/user/recipients").as("recipients");
     cy.visit("/pulse/create");
+    cy.wait("@recipients");
```

Comment:
This is the actual fix.
Everything else is making this test easier to read and getting rid of linter errors.